src/app/stage/stage/stage.component.html

```diff
@@ -24,6 +24,9 @@
     aria-label="Presets"
     mat-icon-button
     matTooltip="Presets"
+    matBadge="{{ presets.length }}"
```

Comment:
1. `[matBadge]="presets.length"`
2. Won't could default - empty preset here, only custom `presets.length - 1`

Reply:
Are you sure with this `presets.length - 1` ? You don't want to count _Default_, do you?
Now with only `presets.length` it looks like this:
![obraz](https://user-images.githubusercontent.com/42266201/70278124-f712a300-17b3-11ea-8189-7504bd9cbe99.png)

Reply:
Yes, you're right, so just update binding